app/vmui/packages/vmui/src/pages/CardinalityPanel/hooks/useCardinalityFetch.ts

```diff
@@ -72,23 +72,23 @@ export const useFetchQuery = (): {
 
     const prevDayParams = {
       ...requestParams,
-      date: dayjs(requestParams.date).subtract(1, "day").tz().format(DATE_FORMAT),
+      date: dayjs(requestParams.date).subtract(1, "day").format(DATE_FORMAT),
     };
 
     const urls = [
       getCardinalityInfo(serverUrl, requestParams),
       getCardinalityInfo(serverUrl, prevDayParams),
     ];
 
-    if (prevDate !== date) {
+    if (prevDate !== date && totalParams.match || totalParams.focusLabel) {
```

Comment:
could you please elaborate on this change? should it be `prevDate !== date && (totalParams.match || totalParams.focusLabel)` instead?

Reply:
Thank you for noticing, there was a mistake here, I've corrected it.

- An additional request is needed to obtain the totals without filtering; this is necessary for `Total series` and `Percentage from total`.
- In cases where there is no filtering, the totals in both requests are identical, and an additional request can be avoided.